build.gradle

```diff
@@ -118,6 +118,7 @@ subprojects {
             // JDepend manages to include a different version of itself in its own transitive closure.
             force "jdepend:jdepend:${project.jdepend.toolVersion}"
         }
+        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
```

Comment:
I think this PR can't be merged without this caching update because this PR already synced up with the latest app-tools-lib that has gone re-organization.